persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -46,36 +46,39 @@ email = excluded.email,
 phone = excluded.phone,
 accent_id = excluded.accent_id,
 team = excluded.team,
-connection_status = excluded.connection_status,
```

Comment:
When updating the connection status we should be using the `upsertUserConnectionStatus` query.